src/state/actionTypes/index.js

```diff
@@ -1,8 +1,7 @@
-export const GET_DATA = 'GET_DATA';
+//These are for filtering/searching data already retrieved from the API
 export const FILTER_SEARCH = 'FILTER_SEARCH';
-export const RESET_DATA = 'RESET_DATA';
-
-//Backend filter & fetch actions
+export const RESET_CASE_DATA = 'RESET_DATA';
+//Actions to choose filters and fetch API data
```

Comment:
I like how you are providing comments throughout the codebase that provides context for our successors.